src/agents/workflows/subgraphs/step1_preprocessing/graph.py

```
@@ -1,53 +1,191 @@
+        if not content:
+            continue
+
+        # 2. 리스트 형태 처리 (텍스트 + 파일 혼합 입력 시 핵심 로직)
+        if isinstance(content, list):
+            for item in content:
+                if isinstance(item, dict):
+                    t = item.get("type", "").lower()
+                    if t in {"image_url", "image", "file", "file_upload", "attachment"}:
+                        # 'file_path' 키를 최우선으로 찾고, 없으면 url/path/name 순으로 탐색
+                        path = item.get("file_path") or item.get("url") or item.get("path") or item.get("name")
+                        if path: return path
```

Ruff (0.14.13)

Multiple statements on one line (colon)

(E701)